Add AGENTS.md and CLAUDE.md, repair issue templates, fix chip size in README (#82)
ultralytics/yolo: AGENTS.md

pip3 install -U -r requirements.txt        # install (numpy, scipy, opencv-python, torch, matplotlib, tqdm, h5py)
python train.py                            # train; edit the train_path values near the top of train.py first
python train.py -resume 1                  # resume from weights/latest.pt
(cd weights && bash download_weights.sh)   # fetch pretrained xView weights
python detect.py -image_folder path/to/train_images
ruff format . && ruff check --fix .        # formatting is applied on PRs by Ultralytics Actions, not a repo config
```

- There is no test suite, no `pyproject.toml`, and no packaging. CI is limited to `format.yml` (Ultralytics Actions), `cla.yml`, and `stale.yml` — no build or test workflow runs on PRs.
- Both scripts use single-dash argparse flags (`-cfg`, `-img_size`, `-batch_size`), not the `--flag` convention of later Ultralytics repos.

## Architecture

- `models.py` — `parse_model_config()` reads a Darknet `cfg/*.cfg` file into module definitions, `create_modules()` builds them, and `Darknet` runs the forward pass with `YOLOLayer` heads. `cfg/c60_a30symmetric.cfg` (60 classes, 30 k-means anchors) is the default for both `train.py` and `detect.py`.

> - `utils/datasets.py` — `ListDataset` iterates full-resolution `.tif` images, samples one augmented chip per image, and reads label boxes from the `utils/targets_c60.mat` MATLAB file; `ImageFolder` serves inference. Images are sampled with per-image weights, and classes with `xview_class_weights()`.

Reviewer: 📝 **LOW**: The architecture guide has the same incorrect one-chip description as the README. `ListDataset.__next__()` deliberately creates eight crop centers and appends eight chips for each selected `.tif`; documenting one chip per image will mislead future changes to this data path.

**Suggested change:**
```suggestion
- `utils/datasets.py` — `ListDataset` samples full-resolution `.tif` images with per-image weights and creates eight augmented chips per selected image; it reads label boxes from the `utils/targets_c60.mat` MATLAB file. `ImageFolder` serves inference.
```
